scripts/build/Dockerfile.amd-rocm

```diff
@@ -0,0 +1,97 @@
+FROM rocm/pytorch:rocm4.0.1_ubuntu18.04_py3.6_pytorch
+
+ARG CC=gcc
+
+# Environment
+ENV BRANCH=$BRANCH
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8
+ENV LANGUAGE=en_US.UTF-8
```

Comment:
The `ENV` instruction supports multiple variables to be set at one time. This would reduce the number of container image layers.
```dockerfile
ENV BRANCH=$BRANCH \
    DEBIAN_FRONTEND=noninteractive \
    LC_ALL=en_US.UTF-8 \
    LANG=en_US.UTF-8 \
    LANGUAGE=en_US.UTF-8
```

Reply:
OK